app/views/posts/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container">
+  <div class="user_info">
+    <div class="img">
+      <%= image_tag @user.photo, alt: "User Photo", class: "user-photo"%>
+    </div>
+    <div class="user">
+      <h2><%= @user.name %></h2>
+      <p><strong>Number of posts: </strong><%= @user.posts.count %></p>
+    </div>
+  </div>
+<a href=""></a>
+  <% @posts.limit(2).each_with_index do |post, index| %>
+    <div class="post">
+      <h3> <%= link_to "Post", user_post_path(@user, post) %> <%= index+1 %> </h2>
+      <p><%= post.text %></p>
+      <p class='count'>Comments: <%= post.comments.count %> | Likes: <%= post.likes.count %></p>
+    </div>
+    <div class="comments">
+      <% post.recent_comments.each do |comment| %>
+       <p><strong><%= comment.user.name %> :</strong> <%= comment.text %></p>
+      <% end %>
+    </div>
+    <% end %> 
+    <%= link_to "Pagination", user_posts_path(@user), class: "button" %>
+</div>
```

Comment:
- [ ] Kindly ensure the elements match the screen width coverage as provided in [the wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png).

  ![image](https://github.com/Obimbo07/tdn/assets/36542986/7e82f955-6739-45b5-ba8c-59084c673aa9)

Reply:
Fixed.
![image](https://github.com/Obimbo07/tdn/assets/93598189/859a7f8c-24fe-4eb1-8b8a-852013cb86e2)